content/en/docs/polo-features/emojis.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Emojis
+weight: 6
+description: What to the emojis in the log mean?
+
+---
+
+## What do the emojis mean?
+
+<p>Every now and then you will see an emoji within the Callsign Info Panel, for example if you enter KI2D you'll get the following: 🤩 Sebastián - Ham2K PoLo Creator.</p> 
+
+You can expect to see any of the following:<br>
```

Comment:
I'm not sure it's a good idea to provide a full index of all the emoji, specially because for most of them, the emoji is shown along a text explaining what it means (except for youtubers and tiktokers and instagrams, but I'll change that in the notes file).

Also, I rather not provide a link to the file itself. I want the emoji to be a bit of a surprise when you see them

On the other hand, we might want to include some text here explaining that this is connected to the Call Notes feature, and explaining that you can disable it if you want, and how.

Reply:
OK I'll make those changes.